docs/code/MlNetCookBook.md

```diff
@@ -0,0 +1,212 @@
+# ML.NET Cookbook
+
+This document is intended to provide essential samples for common usage patterns of ML.NET. 
+It is advisable to be at least minimally familiar with [high-level concepts of ML.NET](MlNetHighLevelConcepts.md), otherwise the terminology in this document may be foreign to you.
+
+## How to use this cookbook
+
+Developers often work by copying and pasting source code from somewhere and then adapting it to their needs. We do it all the time.
+
+So, we decided to embrace the pattern and provide an authoritative set of example usages of ML.NET, for many common scenarios that you may encounter.
+These examples are multi-purpose:
+
+- They can kickstart your development, so that you don't start from nothing,
+- They are annotated and verbose, so you have easier time adapting them to your needs.
+
+Each sample also contains a snippet of the data file used in the sample. We mostly use snippets from our test datasets for that.
```

Comment:
These still assume plenty of background knowledge, like which `using` statements are needed. To handle some of the 100-level knowledge, we should link to/include a basic setup/getting-running guide. We could link to a fully fleshed-out example w/ a full Visual Studio solution in the example.

I'm thinking similar to these:
https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis
https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/BinaryClassification_SentimentAnalysis

Reply:
Thanks for your comment! I do not plan to cover the basic VS setup etc. 
Please feel free to author these examples if you want.